Makefile

```diff
@@ -12,10 +12,13 @@ ifneq ($(shell uname -s),Darwin)
 endif
 
 build-all:
-	go build -tags containers_image_openpgp ./...
+	go build -tags aro,containers_image_openpgp ./...
 
 aro: generate
-	go build -tags containers_image_openpgp,codec.safe -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(COMMIT)" ./cmd/aro
+	go build -tags aro,containers_image_openpgp,codec.safe -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(COMMIT)" ./cmd/aro
+
+runlocal-rp:
+	go run -tags aro ./cmd/aro rp
```

Comment:
Can we do something like `go run -tags aro ./cmd/aro %CMD`? So we do `make runlocal rp` or `make runlocal monitor`?

Reply:
This is not a Makefile feature :-(